### PR TITLE
custom configuration dictionary

### PR DESCRIPTION
**Pull request by ransford**
Makes it easier to provide custom reader configuration. Shouldn't break anything.

### CODE REVIEW

**Comment by ransford** [comment]
Great idea! I'll do that.

**Comment by ransford** [comment]
Done and tested. Look good @ukrutt?

File: sllurp/llrp.py [hunk 1]
```diff
@@ -171,6 +191,7 @@ def __init__(self, factory, duration=None, report_every_n_tags=None,
                  mode_identifier=None,
                  session=2, tag_population=4,
                  tag_filter_mask=None,
+                 config_dict=DEFAULT_READER_CONFIG,
```
Comment (ukrutt) [inline sllurp/llrp.py]: How about moving it even one step further out, to `LLRPClientFactory`?

Comment (ransford) [reply sllurp/llrp.py]: Oh, great idea!

Comment (ransford) [reply sllurp/llrp.py]: Done in e4a6da0.

**Comment by ransford** [comment]
Latest changes LGTM, though CircleCI seems stuck.

**Pull request merged**